docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,222 @@
+```
+pixi global uninstall python pip
+```
+
+### Injecting dependencies
+
+Create environment `python`, install package `python` and expose all binaries of that package.
+Then add package `hypercorn` to environment `python` but doesn't expose its binaries.
+
+```
+pixi global install python
+pixi global inject --environment=python hypercorn
+```
+
+Update package `cryptography` (a dependency of `hypercorn`) in environment `python`
+
+```
+pixi update --environment python cryptography
+```
+
+Then remove `hypercorn` again.
+```
+pixi global remove --environment=python hypercorn
+```
+
+
+### Specifying which binaries to expose
+
+Make a new environment `python_3_10` with package `python=3.10` and no exposed binaries
+```
+pixi global install --environment python_3_10 --expose "python3.10=python" python=3.10
+```
+
+Now `python3.10` is available.
+
+
+Run the following in order to expose `python` from environment `python_3_10` as `python310` instead.
+
+```
+pixi global expose --environment python_3_10 --package python "python310=python"
+```
+
+Now `python310` is available, but `python3.10` isn't anymore.
+
+!!! note
+
+    It should be possible to infer `--package`, let's discuss if there are edge cases to consider.
+
+### Syncing
+
+Most `pixi global` sub commands imply a `pixi global sync`.
+
+- Users should be able to change the manifest by hand (creating or modifying (adding or removing))
+- Users should be able to "export" their existing environments into the manifest, if non-existing.
+- The manifest is always "in sync" after `install`/`remove`/`inject`/`other global command`.
+
+
+First time, clean computer.
+Running the following creates manifest and `~/.pixi/envs/python`.
+```
+pixi global install python
+```
+
+Delete `~/.pixi` and syncing, should add environment `python` again as described in the manifest
+```
+rm `~/.pixi`
+pixi global sync
+```
+
+If there's no manifest, but existing environments, pixi will create a manifest that matches your current environments.
+It is to be decided whether the user should be asked if they want an empty manifest instead, or if it should always import the data from the environments.
+```
+rm <manifest>
+pixi global sync
+```
+
+If we remove the python environment from the manifest, running `pixi global sync` will also remove the `~/.pixi/envs/python` environment from the file system.
+```
+vim <manifest>
+pixi global sync
+```
+
+## Open Questions
+
+### Should we version the manifest?
+
+Something like:
+
+```
+[manifest]
+version = 1
+```
+
+We still have to figure out which existing programs do something similar and how they benefit from it.
+
+### Multiple manifests
+
+We could go for one default manifest, but also parse other manifests in the same directory.
+In order to modify those with the `CLI` one would have to add an option `--manifest` to select the correct one.
+
+- pixi-global.toml: Default
+- pixi-global-company-tools.toml
+- pixi-global-from-my-dotfiles.toml
```

Comment:
Similar to `.env` I would use a dot seperator. 

```suggestion
- pixi-global.company-tools.toml
- pixi-global.from-my-dotfiles.toml
```

Reply:
Sounds good to me!